pygmt/src/coast.py

```diff
@@ -20,7 +23,6 @@
     A="area_thresh",
     B="frame",
     C="lakes",
-    D="resolution",
```

Comment:
After removing `D="resolution"`, this alias won't be shown in the alias list. Need to wait for #3945.